main/src/kotlinx/team/infra/Publishing.kt

```diff
@@ -33,6 +33,11 @@ open class PublishingConfiguration @Inject constructor(val objects: ObjectFactor
     fun include(vararg name: String) {
         includeProjects.addAll(name)
     }
+
+    /**
+     * Build all artifacts for publishing and deploy them from a single mac agent.
+     */
+    var singleAgentMacDeployment: Boolean = false
```

Comment:
Perhaps, it'd be better to place it in `teamcity` DSL block because it affects generated teamcity build configurations?
On the other hand, `libraryStagingRepoDescription` was recently moved (see the commit 25e365088a) from `teamcity` to `publishing/sonatype` block, because while it also affects TC configurations, it is semantically closer to sonatype publishing.
WDYT?

Reply:
I think "publishing" DSL is the appropriate place for the property. It affects publishing, even if only in TeamCity.